CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 15.5.4 (April 11, 2017)
+
+* Fix compatibility with Enzyme by exposing `batchedUpdates` on shallow renderer. [See #9382](https://github.com/facebook/react/commit/69933e25c37cf5453a9ef132177241203ee8d2fd).
```

Comment:
Here, we try to separate React, React DOM, and React Test Renderer changes.
Can we make it clearer which package each change corresponds to?

I also think it might be simpler to roll addon updates right into this list, and summarize them as React Addons bullet point list for each version instead of giving each addon a section.